content/blog/2018_cfp_announcement.md

```diff
@@ -0,0 +1,79 @@
+Title: Call For Proposals is open!
+Date: 2018-04-30 3:28
+Category: 2018
+Slug: announcing-cfp-2018
+Summary: PyCon India 2018 - Call For Proposals is open!
+
+
+The time has come! And we are thrilled to announce the opening of Call for Proposals. We encourage everyone to [submit a proposal](https://in.pycon.org/cfp/pycon-india-2018/proposals/) on any topic they would like to share with the community. 
+
+Here are some of the important dates to keep in mind:
+
+- **1st July**: Proposal submission deadline
+- **15th July**: Presentation upload
+- **4th August**: Talk acceptance
+- **15th August**: Fianl schedule
```

Comment:
Spelling correction `Final`

Reply:
Need dates as well. If dates are correct then I can just correct spelling and push the changes.

Reply:
https://mail.python.org/pipermail/inpycon/2018-April/012041.html
does not have any mention of presentation upload date.

Reply:
Ah I am not sure if there is a separate presentation upload date. Need to verify this with CFP team.

Reply:
I picked it up from here: https://in.pycon.org/blog/2018/pycon-india-2018-reschedule.html
But in the update I'm about to push, I'm including only the points mentioned in CFP thread on our mailing list.